server.js

```diff
@@ -90,11 +92,19 @@ server.get('/auth/facebook/callback',
          passport.authenticate('facebook', { failureRedirect: '/login' }),
          function(req, res) {
   console.log('authentication passed, redirecting to home page');
-  res.redirect('/home');
+  res.redirect('/');
 });
 
 server.get('/login', ensureAuthenticated, function (req, res) {
-    res.redirect('/home');
+    res.redirect('/');
+});
+
+server.get('/', function(req, res){
```

Comment:
This may be a little funky with our deployed setup.  It may pay to try it out there before merging to master.  Check out the directions in the slack.

Nitpick - formatting of this block